src/DynamoCoreWpf/NodeViewCustomization/NodeViewCustomizationLoader.cs

```diff
@@ -1,10 +1,11 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
 using Dynamo.Logging;
 using Dynamo.Utilities;
 using Dynamo.Wpf.Properties;
+using TypeExtensions = Dynamo.Utilities.TypeExtensions;
```

Comment:
whats this for?

Reply:
This is because TypeExtensions was ambiguous with System.Reflection.TypeExtensions